test262/index.html

```diff
@@ -115,6 +115,8 @@ <h2>test262 parser tests</h2>
     <script src="https://unpkg.com/luxon@1.27.0"></script>
     <script src="https://unpkg.com/chart.js@3.3.2"></script>
     <script src="https://unpkg.com/chartjs-adapter-luxon@1.0.0"></script>
+    <script src="https://cdn.jsdelivr.net/npm/hammerjs@2.0.8"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/chartjs-plugin-zoom/1.0.1/chartjs-plugin-zoom.min.js" integrity="sha512-b+q5md1qwYUeYsuOBx+E8MzhsDSZeoE80dPP1VCw9k/KA9LORQmaH3RuXjlpu3u1rfUwh7s6SHthZM3sUGzCkA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
```

Comment:
Can we get those from unpkg as well? Seems weird to mix three sources for external JS :man_shrugging:

Reply:
The place I was looking at (cdnjs, I think) only listed cloudflare; so I thought that was the only available place

Reply:
Nah, unpkg makes anything on npm available to the browser :)

We probably want [`https://unpkg.com/hammerjs@2.0.8/hammer.min.js`](https://unpkg.com/hammerjs@2.0.8/hammer.min.js) (default is not minified) and [`https://unpkg.com/chartjs-plugin-zoom@1.0.1`](https://unpkg.com/chartjs-plugin-zoom@1.0.1)

Reply:
Evidence number two that I'm a noob at being a webdev :P 
I switched it to your suggested urls.